oeps/best-practices/oep-0049-django-app-patterns.rst

```diff
@@ -267,7 +267,31 @@ Do note that even if you expose your tasks through ``api.py`` to be used by othe
 still be configured with the ``tasks`` import name, as the celery identifier for your task (as set by the celery
 decorator) is based off the original file.
 
+docs/decisions/0001-purpose-of-this-repo.rst or 0001-purpose-of-this-app.rst
+============================================================================
+
+This should be an architectural decision record (ADR) describing the decision behind adding the app. Future ADRs should also be placed in this directory.
```

Comment:
Can you add a link to https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html#adrs?

Reply:
[fixed]